catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/bower.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "catalog-admin-module-registry",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "backbone-relational": "0.8.8",
```

Comment:
You don't need both relational and associations. Which one are you actually using?